arviz/plots/kdeplot.py

```diff
@@ -162,12 +168,13 @@ def plot_kde(
         ax.plot(x, density, label=label, **plot_kwargs)
         if rotated:
             ax.set_xlim(0, auto=True)
-            rug_x, rug_y = np.zeros_like(values), values
+            rug_x, rug_y = np.zeros_like(values) - rug_space, values
         else:
             ax.set_ylim(0, auto=True)
-            rug_x, rug_y = values, np.zeros_like(values)
+            rug_x, rug_y = values, np.zeros_like(values) - rug_space
 
         if rug:
+            del rug_kwargs["space"]
```

Comment:
There is a better way to do this? I am deleting `space`, because `ax.plot`, does not have a space keyword